packages/shared/details-page/DetailsPage.tsx

```diff
@@ -52,12 +52,27 @@ type DetailsPageTitleProps = {
 export const DetailsPageTitle: React.FC<DetailsPageTitleProps> = ({
   resource,
   resourceModel,
-}) => (
-  <span>
-    <ResourceIcon resourceModel={resourceModel} />
-    {getName(resource)}
-  </span>
-);
+}) => {
+  const { t } = useCustomTranslation();
+
+  const showOwnerRef = resource?.metadata?.ownerReferences;
+  return (
+    <div className="odf-details-title">
+      <div>
+        <ResourceIcon resourceModel={resourceModel} />
+      </div>
+      <div className="odf-title-owner-reference">
+        {getName(resource)}
+        {showOwnerRef && (
```

Comment:
```suggestion
        {!!showOwnerRef?.length && (
```

corner case is it will be true for an empty array

Reply:
I checked it won't come empty in YAML, Empty field is ignored automatically. so we are good

Reply:
cool... thanks !!